video/external/src/backend.rs

```diff
@@ -25,6 +25,11 @@ enum ProxyOrStream {
     Owned(VideoStream),
 }
 
+struct OpenH264Data {
+    download_filename: &'static str,
+    download_sha256: &'static str,
+}
```

Comment:
Is the benefit of this purely code readability/clarity, or something more technical too? The former is also a fine reason to have it, just curious!

Reply:
Also, maybe instead of adding the `download_` prefix to both fields, maybe the struct name could carry this information...? But it's not `pub`, so it doesn't matter much...

Reply:
This change is a bit future-oriented, as I will add `local_filenames` to this struct in the next commit, and that's the real motivation of this struct :sweat_smile: